src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -211,12 +195,12 @@ export default function MissingFileOverview({
   mailPreamble,
   officeName,
   requiredDescription,
-  optionalDescription,
   faxNum,
   showConsent,
   allPages,
   fileNameMap,
   requiredFiles,
+  listNameKey,
```

Comment:
This param lets us override the default name property of `applicantName`, so that if we want to use this component in a future form that maybe doesn't have an applicant array but still needs to display uploads a user missed, we can tweak the name of the array property we want to look at.